platform/ios/src/MGLMapView.mm

```diff
@@ -1556,18 +1556,20 @@ - (void)handleDoubleTapGesture:(UITapGestureRecognizer *)doubleTap
     if (doubleTap.state == UIGestureRecognizerStateEnded)
     {
         MGLMapCamera *oldCamera = self.camera;
-        
+
+        double newZoom = round(self.zoomLevel) + 1.0;
```

Comment:
Can you make the corresponding changes to the Zoom In and Zoom Out buttons in the Android and macOS SDKs?

https://github.com/mapbox/mapbox-gl-native/blob/e6c15d0f6f285bc604c0d93381b9b1d3957cb5c9/platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/Transform.java#L180-L184
https://github.com/mapbox/mapbox-gl-native/blob/e6c15d0f6f285bc604c0d93381b9b1d3957cb5c9/platform/macos/src/MGLMapView.mm#L1045

/cc @tobrun